server/routes.py

```diff
@@ -0,0 +1,27 @@
+from flask import Flask, request, Response
+import db
+
+app = Flask(__name__)
+
+#connect to the database
+conn = db.init_db()
```

Comment:
Should we use a less ambiguous abbreviation? could we just use `database` or something explicit?